src/Stryker.Core/Stryker.Core/Options/StrykerOptions.cs

```diff
@@ -17,13 +17,24 @@ public class StrykerOptions
 
         public int AdditionalTimeoutMS { get; }
 
-        public StrykerOptions(string basePath, string reporter, string projectUnderTestNameFilter, int additionalTimeoutMS, string logLevel, bool logToFile)
+        public int ThresholdBreak { get; }
```

Comment:
I'd prefer to store the Threshold as an object on the `StrykerOptions` class with separate properties. I would also expect this to be defined as a JSON object if you use a config file.